packages/wdio-cli/src/interface.js

```diff
@@ -63,12 +65,18 @@ export default class WDIOCLInterface extends EventEmitter {
     /**
      * clear job from interface
      */
-    clearJob ({ cid, passed }) {
+    clearJob ({ cid, passed, retries }) {
         this.jobs.delete(cid)
-        this.result.finished++
+        const retry = !passed && retries > 0
+        if (!retry) {
+            this.result.finished++
+        }
 
         if (passed) {
             this.result.passed++
+        } else if (retry) {
+            this.totalWorkerCnt++
+            this.result.retries++
```

Comment:
shouldn't the amount of retries left, go down instead of increase?

Reply:
This is the amount of times a retry has been run, not the amount of retries remaining. If the above comment was clearer, this probably would have been too :)